recipes/featurewiz/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "featurewiz" %}
+{% set version = "0.1.55" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/featurewiz-{{ version }}.tar.gz
+  sha256: 86ea12944225d92120edf1982268dd51c891ff8901590fb5abeb1a12d693db20
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7, <3.8
```

Comment:
It is very likely that this is only a lower bound as the description says that most python versions work. I would ask upstream and otherwise leave it at `python >=3.7`.